CarpetCleaning/StoredProcedures/Dbo.Loadexceldata.sql

```diff
@@ -0,0 +1,93 @@
+CREATE PROCEDURE DBO.JobDetails_LoadExcelData
+AS
+BEGIN
+
+WHILE ((SELECT COUNT(*) FROM ExcelData WHERE LoadDataNotes = 'Passed Validation') > 0)
+BEGIN
+
+	DECLARE @ThisRowId INT, 
+			@AddressID INT,
+			@JobID INT
+
+	DECLARE @CarpetCleaningServiceTypeId INT = 1,
+			@UrineTreatmentServiceTypeId INT = 2,
+			@ProtectantServiceTypeId INT = 3,
+			@UpholsteryServiceTypeId INT = 7,
+			@AirDuctsServiceTypeId INT = 9,
+			@TileServiceTypeId INT = 12,
+			@DryerVentServiceTypeId INT = 11,
+			@OtherServiceTypeId INT = 18
+				
+		--Inserts FROM the top row and works its way down.
+	SELECT TOP 1 @ThisRowId = RowId FROM DBO.Exceldata WHERE LoadDataNotes = 'Passed Validation'
+	
+	BEGIN TRANSACTION
```

Comment:
You begin the transaction here, but then you don't do anything with it.  You need to check if there were any errors and rollback the transaction.  Or if there were no errors then you can commit the transaction.  Let me know if you need any help with that, but you should be able to look quite a bit of it up.